spec/generators/rspec/controller/controller_generator_spec.rb

```diff
@@ -4,9 +4,10 @@
 
 RSpec.describe Rspec::Generators::ControllerGenerator, type: :generator do
   setup_default_destination
+  it_behaves_like "a request spec generator"
```

Comment:
Oh I see the other discussion now.... Not to crush your spirit here, but may I suggest to split the two behaviour changes (file naming & content of ControllerGenerator? The ControllerGenerator and the Request/Integration Generator are **not** the same thing.
There is no inheritance or module connection at all, sometimes repetition in tests is a good thing for understandability

ControllerGenerator executes also the rails generators, while the RequestGenerator **only** generates the test files (for example to test already existing untested controllers in a codebase)

Reply:
> The ControllerGenerator and the Request/Integration Generator are not the same thing.

My idea here is not make them the same thing, but enforce them to behave as a "request generator". That way, given a certain configuration we can expect them to generate the same request specs.

> There is no inheritance or module connection at all, sometimes repetition in tests is a good thing for understandability.

In the case of the controller generator, there is no inheritance, but they share the same template file.

I prefer to use the shared example instead of repeat test, because I like the idea of make sure that this compatibility is maintained in new versions, so the file naming convention becomes a specification for requests generators.